src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,38 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/car/driver/add")
```

Comment:
<img width="627" alt="image" src="https://user-images.githubusercontent.com/61326568/191007961-b216fdd4-9722-4c2a-96be-9c579231a892.png">
use the plural, as here, check all URLs